### DISCUSSION TITLE
Current project status?

### DISCUSSION THREAD

**Discussion by nordborn** [AbstractMachinesLab/caramel#102] [General] [answered]
Hi everyone, and especially Leandro! It seems that the dev process was interrupted... IMO, the project is very promising and it's sad to see that it is not developing(... What do you think about further steps?

**Comment by leostera** [comment, marked as answer]
Hi! 👋🏽 sorry for the super late response, I totally missed this discussion topic. 

Unfortunately my focus has shifted to other projects and I don't think I'll continue working on Caramel any time soon.

This may change if in the near future I free up considerable time, but I can't make any promises.  If I do find the time it's likely that I'll do another rewrite, starting off from the last rewrite in the `sugarcane` branch, and the language would move further away from OCaml.

In the meantime, I encourage you to try out [Gleam](https://gleam.run) ✨ -- Louis is doing a wonderful job with it and there are plenty of people using it for real-life workloads already.
